Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp

```diff
@@ -8403,7 +8407,8 @@ void SpeculativeJIT::cageTypedArrayStorage(GPRReg baseReg, GPRReg storageReg, bo
 {
     auto untagArrayPtr = [&]() {
 #if CPU(ARM64E)
-        m_jit.untagArrayPtrLength64(MacroAssembler::Address(baseReg, JSArrayBufferView::offsetOfLength()), storageReg, validateAuth);
+        // FIXME: It should be caged with maxByteLength.
```

Comment:
Is this FIXME still needed?  You're already using `JSArrayBufferView::offsetOfMaxByteLength()` for the caging here, right?

Reply:
Dropped.